docker-compose.test.yml

```diff
@@ -9,3 +9,9 @@ services:
       - POSTGRES_HOST_AUTH_METHOD=trust
     ports:
       - "5432:5432"
+    volumes:
+      - ./database/schema.sql:/docker-entrypoint-initdb.d/1-schema.sql
```

Comment:
Use the schema file to init the Postgres database followed [this example](https://gist.github.com/onjin/2dd3cc52ef79069de1faa2dfd456c945)